tokens/connect/trezor.md

```diff
@@ -19,49 +19,49 @@ description: 通过本教程学习如何利用Moonbeam兼容以太坊的特性
 
 ## 查看先决条件 {: #checking-prerequisites } 
 
-在您开始之前，请将您的[Trezor Suite](https://suite.trezor.io/)更新至最新的可用版本。 与此同时，请确认您的Trezor硬件钱包运行的是最新的固件。Trezor wiki有提供如何更新[Trezor One](https://wiki.trezor.io/User_manual:Updating_the_Trezor_device_firmware__T1)和[Trezor Model T](https://wiki.trezor.io/User_manual:Updating_the_Trezor_device_firmware)设备固件的教程。
+在您开始之前，请将您的[Trezor Suite](https://suite.trezor.io/){target=_blank}更新至最新的可用版本。 与此同时，请确认您的Trezor硬件钱包运行的是最新的固件。Trezor wiki有提供如何更新[Trezor One](https://wiki.trezor.io/User_manual:Updating_the_Trezor_device_firmware__T1){target=_blank}和[Trezor Model T](https://wiki.trezor.io/User_manual:Updating_the_Trezor_device_firmware){target=_blank}设备固件的教程。
 
 截至本教程撰写时，使用的版本如下：
 
  - Trezor Suite 21.5.1
  - Trezor One firmware v1.10.0
  - Trezor Model T firmware v2.4.0
 
-除此之外，您还需要MetaMask作为您Trezor设备与Moonbase Alpha之间的媒介，请确保您的[MetaMask已成功连接至Moonbase Alpha](/integrations/wallets/metamask/)。请注意，您的Trezor设备会在您连接至MetaMask的网络内签署交易。
+除此之外，您还需要MetaMask作为您Trezor设备与Moonbase Alpha之间的媒介，请确保您的[MetaMask已成功连接至Moonbase Alpha](/integrations/wallets/metamask/){target=_blank}。请注意，您的Trezor设备会在您连接至MetaMask的网络内签署交易。
 
 ## 将您的Trezor账户导入MetaMask {: #importing-your-trezor-account-to-metamask } 
 
 首先，您需要设置一个钱包（标准或是隐藏钱包）。当您成功连接至您的Trezor设备，请将其解锁，并在Trezor Suite获得一个钱包设置。接着，将您的Trezor以太坊账户导入MetaMask，您可以跟随以下步骤进行操作：
 
  1. 点击右上角的logo展开使用菜单
- 2. 选取“连接硬件钱包“
+ 2. 选取**连接硬件钱包**
 
 ![MetaMask Connect Hardware Wallet](/images/tokens/connect/ledger/ethereum/ledger-2.png)
 
 进入下个页面之后，选择您要在MetaMask使用的硬件钱包。截至本教程撰写时，仅支持Ledger和Trezor硬件钱包。您可以根据以下步骤进行操作：
 
  1. 点选Trezor logo
- 2. 点击“继续”
+ 2. 点击**继续**
 
 ![MetaMask Select Trezor Hardware Wallet](/images/tokens/connect/trezor/trezor-2.png)
 
-点击按钮之后，将会出现一个名为`TrezorConnect`的页面，您需要在这里配对您的设备。如果您已经打开Trezor Suite且设备已经连接成功，则不需要此步骤。在这里，请点击“配对设备”。
+点击按钮之后，将会出现一个名为**TrezorConnect**的页面，您需要在这里配对您的设备。如果您已经打开Trezor Suite且设备已经连接成功，则不需要此步骤。在这里，请点击**配对设备**。
 
 ![Trezor Hardware Wallet Connect Pair Device](/images/tokens/connect/trezor/trezor-3.png)
 
 跳转到下个页面，您可以跟随以下步骤操作：
 
- 1. 点击“检查设备”，这将会打开一个新菜单显示您想要连接的Trezor设备（若可用）
+ 1. 点击**检查设备**，这将会打开一个新菜单显示您想要连接的Trezor设备（若可用）
  2. 选取您想要使用的Trezor设备
- 3. 点击“连接”
+ 3. 点击**连接**
 
 ![Trezor Hardware Wallet Connect Wizard Select and Connect Device](/images/tokens/connect/trezor/trezor-4.png)
 
-当您的设备已成功连接，您需要允许MetaMask读取其公钥。因此，请点击“允许本次使用“。您也可以勾选“不要再询问我”选项。
+当您的设备已成功连接，您需要允许MetaMask读取其公钥。因此，请点击**允许本次使用**。您也可以勾选**不要再询问我**选项。
```

Comment:
the `**"whatever"**` items shouldn't be translated 

should be **Allow once for this session**. 

and then  **Don't ask me again**